### PR TITLE
Fix github actions workflow

### LINKED ISSUE #5: [tracking] Integrate GitHub actions and use jaegertracing dockerhub/quay account
- [ ] Integrate with GitHub actions
- [ ] As of #4, the project uses `annanay25:jaeger-docker-protobuf:latest` image. Move to the jaegertracing user on dockerhub/quay

### PR DESCRIPTION
**Pull request by annanay25**
Fixes #5 

@yurishkuro - Could you please set the dockerhub key as a secret in this repo?

Once that is set we should have automated builds set up.

Signed-off-by: Annanay <annanayagarwal@gmail.com>

### CODE REVIEW

File: .github/workflows/dockerimage.yml [hunk 1]
```diff
@@ -18,29 +17,25 @@ jobs:
...
         echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin
 
         function tag_and_push {
-          docker tag build "thethingsindustries/protoc:${1}" && docker push "thethingsindustries/protoc:${1}"
+          docker tag build "jaegertracing/protobuf:${1}" && docker push "jaegertracing/protobuf:${1}"
         }
 
         if [[ "${GITHUB_REF}" == "refs/heads/master" ]]; then
-          tag_and_push "development"
+          tag_and_push "master"
```
Comment (yurishkuro) [inline .github/workflows/dockerimage.yml]: should this be 'latest'?

**Review by yurishkuro — APPROVED** [review]
I added DOCKER_PASSWORD and DOCKER_USERNAME secrets to the repo.

File: .github/workflows/dockerimage.yml (hunk 1 above)
Comment (annanay25) [reply .github/workflows/dockerimage.yml]: Done.

**Comment by annanay25** [comment]
Huh, DCO is going to show an error because past commits are not signed? @yurishkuro

**Comment by yurishkuro** [comment]
can you squash them?

**Comment by annanay25** [comment]
There's a ton of conflicts, I'll take a stab at this tomorrow.

**Comment by annanay25** [comment]
Hi @yurishkuro - There's a merge conflict every 3-4 commits and there's ~260 of them 😥 

Beginning to question if we need past commits to be signed?

**Comment by yurishkuro** [comment]
You only have two commits in this PR, how can there be so many conflicts?

The signing of commits is a CNCF requirement.

**Comment by annanay25** [comment]
All of my commits are signed. Merging this.

**Pull request merged**